docs/coordinates/velocities.rst

```diff
@@ -312,9 +312,9 @@ the same: the radial velocity should be essentially the same in both frames:
     plt.plot_date(time.plot_date, gcrs.radial_velocity.to(u.km/u.s))
     plt.ylabel('RV [km/s]')
 
-but this result is clearly nonsense, with values from -1000 to 1000 km/s.  The
+But this result is clearly nonsense, with values from -1000 to 1000 km/s. The
```

Comment:
```suggestion
But this result is nonsense, with values from -1000 to 1000 km/s instead of the ~10 km/s we expected. The
```
I think I wrote this originally but on reflection I'm trying to eliminate "clearly" from my vocabulary...

I can also do this as a separate simple commit since it will muck up the line-lengths.

Reply:
I'll make the change! I have a few other separate commits for this package.